godelegator/templates/sa.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.global.secretManager -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.deployment.name }}-{{ .Values.godelegator.serviceAccount.name }}
+  labels: {{ .Values.deployment.name }}
+  # TODO: This annotation should be sync with GCP SA email injected from GH actions
```

Comment:
@nick-bisonai Can you please explain about this TODO? Is it relevant or coming from other templates?

Reply:
@jay-kim-kor Do you know about this TODO?

Reply:
I added that TODO during I was integrating gsm. Because we have to execute command when we want to change or integrate new service account with k8s apps.

But, that is added temporarily, I'll move it to issue now. 😄